src/core/js/views/questionView.js

```diff
@@ -39,7 +39,19 @@ define([
       this._runModelCompatibleFunction('setupDefaultSettings');
       // Blank method for setting up questions before rendering
       this.setupQuestion();
+    }
+
+    /** @type {boolean} */
+    get canShowMarking() {
+      if (!this.model.get('_onlyShowFinalMarking')) {
+        return this.model.get('_canShowMarking');
+      }
+
+      const attemptsLeft = this.model.get('_attemptsLeft');
+      const hasMultipleAttempts = Boolean(this.model.get('_attempts') > 1);
```

Comment:
With:
```js
Boolean(this.model.get('_attempts') > 1);
```

Use either:
```js
(this.model.get('_attempts') > 1) // comparison operator gets evaluated to a boolean
```

or

```js
Boolean(this.model.get('_attempts')) // boolean coercion, false when value is 0, null, undefined, "", etc, true otherwise
```

They do the same thing. I think I like the comparison operator better.
```js
(this.model.get('_attempts') > 1)
```